### DISCUSSION TITLE
Failed to resolve import "cmdk" from "src/components/ui/command.tsx". Does the file exist?

### DISCUSSION THREAD

**Discussion by ManishKarki1997** [shadcn-ui/ui#4080] [Q&A]
Hello.

I ran this command 

`npx shadcn-ui@latest add command
`

And copy/pasted the code from here https://ui.shadcn.com/docs/components/command#dialog

I get this error now, i'm not sure why. I've even installed cmdk manually, removed node_modules etc. 

I've also attached a screenshot below 
![image](https://github.com/shadcn-ui/ui/assets/54467933/ab8b400e-ebce-422e-a17e-e5cebc59a8ce)

Any idea what the cause might be?

Thanks

**Comment by ManishKarki1997** [comment]
Nevermind, it's probably because of misconfiguration in my docker configuration. I ran with normal npm run dev and it works.

**Comment by harshau9** [comment]
even i have this issue not sure why
